aspnetcore/razor-components/get-started.md

```diff
@@ -5,58 +5,167 @@ description: Learn how to get started with the Razor Components framework.
+
+When the app is run, multiple pages are available from tabs in the sidebar:
+
+* Home
+* Counter
+* Fetch data
+
+On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing JavaScript, but Razor Components provides a better approach using C#.
+
+*WebApplication1.App/Pages/Counter.cshtml*:
+
+[!code-cshtml[](get-started/samples_snapshot/3.x/Counter1.cshtml)]
+
+A request for `/counter` in the browser, as specified by the `@page` directive at the top, causes the Counter component to render its content. Components render into an in-memory representation of the render tree that can then be used to update the UI in a flexible and efficient way.
+
+Each time the **Click me** button is selected:
+
+* The `onclick` event is fired.
+* The `IncrementCount` method is called.
+* The `currentCount` is incremented.
+* The component is rendered again.
+
+The runtime compares the new content to the previous content and only applies the changed content to the Document Object Model (DOM).
+
+Add a component to another component using an HTML-like syntax. Component parameters are specified using attributes or child content. For example, a Counter component can be added to the app's homepage by adding a `<Counter />` element to the Index component.
+
+*WebApplication1.App/Pages/Index.cshtml*:
+
+[!code-cshtml[](get-started/samples_snapshot/3.x/Index1.cshtml?highlight=7)]
+
+Run the app. The homepage has its own counter.
+
+To add a parameter to the Counter component, update the component's `@functions` block:
+
+* Add a property for `IncrementAmount` decorated with the `[Parameter]` attribute.
+* Change the `IncrementCount` method to use the `IncrementAmount` when increasing the value of `currentCount`.
+
+*WebApplication1.App/Pages/Counter.cshtml*:
+
+[!code-cshtml[](get-started/samples_snapshot/3.x/Counter2.cshtml?highlight=4,8)]
+
+Specify an `IncrementAmount` parameter in the Home component's `<Counter>` element using an attribute.
 
-![Blazor app home page](https://msdnshared.blob.core.windows.net/media/2018/04/blazor-bootstrap-4.png)
+*WebApplication1.App/Pages/Index.cshtml*:
 
-## Help & feedback
+[!code-cshtml[](get-started/samples_snapshot/3.x/Index2.cshtml)]
 
-Your feedback is especially important to us during this experimental phase for Blazor. If you run into issues or have questions while trying out Blazor, please let us know!
+Run the app. The homepage has its own counter that increments by ten each time the **Click me** button is selected.
 
-* [File issues on GitHub](https://github.com/aspnet/AspNetCore/issues) for any problems you run into or to make suggestions for improvements.
-* Chat with us and the Blazor community on [Gitter](https://gitter.im/aspnet/blazor) if you get stuck or to share how Blazor is working for you.
+## Support & feedback
```

Comment:
I think we can remove this section. I don't think we want to have support & feedback sections in all our ASP.NET Core topics.